### PR TITLE
[amazonechocontrol] HandlerThermostatController

### PR DESCRIPTION
**Pull request by sonic74**

### CODE REVIEW

**Comment by Skinah** [comment]
1. Please put the name of the addon in square brackets in the title. I edited it for you.
2. You may want to review the use of 'logger.error' as error level is reserved for when a critical error occurs that prevents openHAB from working correctly. See here.
<https://www.openhab.org/docs/developer/guidelines.html#f-logging>

**Comment by sonic74** [comment]
> 1. Please put the name of the addon in square brackets in the title. I edited it for you.

Thanks!

> 2. You may want to review the use of 'logger.error' as error level is reserved for when a critical error occurs that prevents openHAB from working correctly. See here.
>    https://www.openhab.org/docs/developer/guidelines.html#f-logging

I replaced error() with warn().

File: bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/Connection.java
Comment (J-N-K) [inline bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/Connection.java:1214]: Please revert these changes. The upper part (l. 1186-1191) should be reworked completely. I'll provide a PR for that.

**Review by J-N-K — CHANGES_REQUESTED** [review]
Otherwise LGTM.

File: bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/Connection.java
Comment (J-N-K) [reply bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/Connection.java:1214]: cf. https://github.com/openhab/openhab-addons/pull/9241

Comment (sonic74) [reply bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/Connection.java]: > Please revert these changes.

Reverted.

**Review by J-N-K — APPROVED** [review]
Many thanks.  Just waiting for Jenkins before merge.

**Comment by J-N-K** [comment]
Jenkins fails unrelated

**Pull request merged**
